1-org/modules/centralized-logging/README.md

```diff
@@ -73,7 +75,7 @@ module "logging_logbucket" {
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
 | bigquery\_options | Destination BigQuery options:<br>- dataset\_name: The name of the bigquery dataset to be created and used for log entries.<br>- logging\_sink\_name: The name of the log sink to be created.<br>- logging\_sink\_filter: The filter to apply when exporting logs. Only log entries that match the filter are exported. Default is "" which exports all logs.<br>- expiration\_days: Table expiration time. If null logs will never be deleted.<br>- partitioned\_tables: Options that affect sinks exporting data to BigQuery. use\_partitioned\_tables - (Required) Whether to use BigQuery's partition tables.<br>- delete\_contents\_on\_destroy: If set to true, delete all contained objects in the logging destination. | <pre>object({<br>    dataset_name               = optional(string, null)<br>    logging_sink_name          = optional(string, null)<br>    logging_sink_filter        = optional(string, "")<br>    expiration_days            = optional(number, null)<br>    partitioned_tables         = optional(bool, true)<br>    delete_contents_on_destroy = optional(bool, false)<br>  })</pre> | `null` | no |
```

Comment:
bigquery_options can be removed from the table of input variables because we're removing the bq sink, right?

Reply:
Good catch, the option was not used by the 1-org step anymore but since the BigQuery destination is not part of the recommendation anymore we can also remove it from the auxiliary module.